cumulus/tasks/pdr-status-check/schemas/output.json

```diff
@@ -23,6 +23,14 @@
     },
     "counter": { "type": "integer" },
     "limit": { "type": "integer" },
-    "isFinished": { "type": "boolean" }
+    "isFinished": { "type": "boolean" },
+    "pdr": {
```

Comment:
Please add a comment and make this a required property.

Reply:
done